### ISSUE TITLE
Add a flag to limit stdout of serverless 

### ISSUE THREAD

**Issue by erwinvaneyk** [serverless/serverless#3449]
# This is a (Feature Proposal)

Tl;dr make output of serverless commandline tool more UNIX friendly by adding a flag to limit stdout of serverless.

Related: #3089

## Description
Currently, lots of additional, boilerplate text is returned for most of subcommands in serverless. This is all fine, when using serverless manually (although I, personally, find it more annoying than useful). I understand that the project is aimed at usability for developers, for which the more extensive output might make sense. However, in my use case, using serverless in a more automated way, it would be safe me from a lot of tedious string matching by having a more specific return format.

For example the output/error for a timeout of `serverless deploy` currently looks like this:
```
Error --------------------------------------------------

     Error: connect ETIMEDOUT <ip>:<port>

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless

  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       6.10.2
     Serverless Version: 1.10.2
```
It would be much more practical with regards to piping/further processing the output by returning actual relevant content. Same goes for other subcommands and error responses. I would propose just outputting the error:
```
Error: connect ETIMEDOUT <ip>:<port>
``` 

Thus I propose the following: Add a flag `--short` (or any other name) that removes all of the boilerplate output of the serverless subcommands.

I am okay with adding it myself or letting anyone else pick this up, if you agree with this proposal.

**Comment by HyperBrain** [comment]
A more common flag name would be `--quiet | -q` (used in various applications) or `--nologo` (used by Microsoft for their command line tools)

**Comment by erwinvaneyk** [comment]
@HyperBrain I completely agree with the `-q, --quiet` for the flag name.

**Comment by medikoo** [comment]
I believe this closely related to https://github.com/serverless/serverless/issues/1720 and after it's addressed it could be solved by propagating mentioned above parts via different log levels (error message and stack information is _error_ level log, while "Get Support"  and "Your Environment Information" is _notice_ level log). Having that using `LOG_LEVEL=error` will produce desired short outut

**Comment by medikoo** [comment]
Closing, as it was addressed with recent logs revamp. Default output is very minimal, and all message logging goes to `stderr` where we can control what level of logs to display
